### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by PorismDominicSkinner**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -103,8 +103,8 @@ const footerProps = data;
         <Navigator mainMenu={mainMenu} topMenuId={topMenuId.toString()} />
         <Switch>
             <Route exact path="/" render={() => ( <HomePage homePageProps={homeProps} classname="main" /> )}/>
-            <Route path="/about-page" render={() =>  <About aboutProps={aboutProps} sideMenu={subMenu} styleName="main" /> }/>
-            <Route path="/how-to" render={() =>  <HowPage styleName="main"/> }/>
+            <Route path="/about-open-referral" render={() =>  <About aboutProps={aboutProps} sideMenu={subMenu} styleName="main" /> }/>
+            <Route path="/how-it-works" render={() =>  <HowPage styleName="main"/> }/>
```
Comment (yemijaiyeola) [inline src/App.js:107]: These 2 links currently don't take you to the requested page when clicked

**Review by yemijaiyeola — CHANGES_REQUESTED** [review]
Do the back end pages need updating?

File: src/App.js (hunk 1 above)
Comment (PorismDominicSkinner) [reply src/App.js:107]: I thought they are, I am getting:

image

and: 

image

**Comment by yemijaiyeola** [comment]
Perfect!!!.  Thanks.  Looks like I did not update my local branch initially with the changes.

**Pull request merged**